src/ProjectTemplates/ComponentsWebAssembly.ProjectTemplates/content/ComponentsWebAssembly-CSharp/Client/appsettings.json

```diff
@@ -0,0 +1,29 @@
+{
+  ////#if (IndividualLocalAuth)
+  // "Authority": "https: //login.microsoftonline.com/",
+  // "ClientId": "33333333-3333-3333-33333333333333333",
+  ////#endif
+
+  ////#if (IndividualB2CAuth)
+  // "AzureAdB2C": {
+  // "Authority": "https:////aadB2CInstance.b2clogin.com/qualified.domain.name/MySignUpSignInPolicyId",
+  // "ClientId": "33333333-3333-3333-33333333333333333",
+  // "ValidateAuthority": false
+  // },
+  ////#endif
+
+  ////#if (OrganizationalAuth)
+  // "AzureAd": {
+  // "Authority": "https://login.microsoftonline.com/22222222-2222-2222-2222-222222222222",
+  // "ClientId": "33333333-3333-3333-33333333333333333"
+  // },
+  ////#endif
+
+  "Logging": {
```

Comment:
More of a question for @javiercn \ @SteveSandersonMS - do we think we should have this? Why is it useful for an application user to see logs?

Reply:
Well, it is useful to debug issues in production I would say.

Reply:
As per https://github.com/dotnet/aspnetcore/pull/20790/files#r408324332 it looks like it would cause 4 extra HTTP requests, which doesn't seem to me like an ideal tradeoff considering how most developers won't have much reason to change the defaults, and those that do can still add this config loading behavior in a couple of lines of code. So I would lean towards not having this here.

In the case of the auth-enabled templates though, if people think it's significantly better to be loading the auth provider config from an external file then that's absolutely fine with me on those templates. That's something that developers will need to edit for sure, unlike the log levels.

Reply:
Also I've measured the effect on the brotli-precompressed app size as 41kb.

Reply:
@captainsafia Just to confirm, @javiercn and I discussed this offline. It's a subjective balance to be struck, but we've agreed to err on the side of keeping this behavior opt-in to reduce the app size and number of HTTP requests in the default case.

So is it OK if you don't include the `appsettings.json` files in the default template, don't reference `Microsoft.Extensions.Logging.Configuration` by default, and only have the `appsettings.json` files in the cases where they are being loaded to configure the auth system? Sorry that adds a bit of extra change to this PR!

Reply:
Thanks for running the numbers on the size increase.

I'll go ahead and remove the `AddConfiguration` call and remove the logging-related configs from the app settings files.